suzieq/poller/worker/nodes/node.py

```diff
@@ -1892,6 +1902,10 @@ async def _parse_init_dev_data_devtype(self, output, cb_token) -> None:
             data = output[1]["data"]
             hmatch = re.search(r'\nHostname:\s+(\S+)\n', data)
             if hmatch:
+                # TODO
```

Comment:
Should we need to address this TODO now?

Reply:
Not right now.